src/api/services/enrichment/enrichment.js

```diff
@@ -32,13 +32,16 @@ export const createEnrichmentRule = async ctx => {
 export const getEnrichmentRuleModel = (sourceData, enrichment) => {
     try {
         let rule;
-        if (typeof sourceData === 'string' || (Array.isArray(sourceData) && typeof sourceData[0] === 'string')) {
+        if (
+            typeof sourceData === 'string' ||
```

Comment:
It could be clearer to extract those conditions in function, like :

```js
const sourceIsDirectPath = (sourceData) => {
    return typeof sourceData === 'string' ||
            (Array.isArray(sourceData) && typeof sourceData[0] === 'string');
}

```